lifecycle-operator/apis/options/v1alpha1/keptnconfig_types.go

```diff
@@ -56,7 +57,8 @@ type KeptnConfig struct {
 	Spec KeptnConfigSpec `json:"spec,omitempty"`
 	// unused field
 	// +optional
-	Status string `json:"status,omitempty"`
+	Status          string `json:"status,omitempty"`
+	BlockDeployment bool   `json:"block-deployment,omitempty"`
```

Comment:
this line can be removed, as there already is a `BlockDeployment` property in the spec struct